# msg/AprilTagDetectionArray.msg
Header header
AprilTagDetection[] detections